examples/flows/single-transfer/input-schema.json

```diff
@@ -0,0 +1,39 @@
+{
+    "type": "object",
+    "additionalProperties": false,
+    "required": [
+        "source_endpoint_id",
+        "destination_endpoint_id",
+        "transfer_items"
+    ],
+    "properties": {
+        "source_endpoint_id": {
+            "type": "string"
+        },
+        "destination_endpoint_id": {
+            "type": "string"
+        },
+        "transfer_items": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "additionalProperties": false,
+                "required": [
+                    "source_path",
+                    "destination_path"
+                ],
+                "properties": {
+                    "source_path": {
+                        "type": "string"
+                    },
+                    "destination_path": {
+                        "type": "string"
+                    },
+                    "recursive": {
+                        "type": "boolean"
+                    }
```

Comment:
If this has a default value, would it be okay to add that as:

```js
{
  "type": "boolean",
  "default": true  // or false
}
```

Reply:
I'd prefer to not add a default value since the Flow itself doesn't define those defaults. It just passes the payload onto the Transfer AP and that's what sets default values.

Reply:
Okay great. Thanks for confirming!